components/QuickStartMenu.tsx

```diff
@@ -24,6 +67,62 @@ const QuickStartMenu = ({ className }: QuickStartMenuProps): JSX.Element => {
           </g>
         </svg>
       </button>
+      {/* NOTE:: Dropdown currently absolutely positioned. Revist if used in place other than the navigation */}
+      <div
+        ref={quickStartMenuRef}
+        className={classnames(
+          'shadow-sm',
+          'bg-white',
+          'border',
+          'border-gray-light',
+          'absolute',
+          'top-20',
+          'w-56',
+          'right-1',
+          'lg:transform-none',
+          'lg:transition-none',
+          'transform-gpu',
+          {
+            [quickStartMenuInactiveClasses]: !isOpen,
+            [quickStartMenuActiveClasses]: isOpen,
+          }
+        )}
+      >
+        <div className={classnames('hidden')}>{SitecoreQuickLinks.title}</div>
+        <ul className={classnames('py-3')}>
+          {SitecoreQuickLinks.children?.map((child, index) => (
+            <li
+              key={`child-${index}`}
+              className={classnames(
+                'font-semibold',
+                'hover:bg-teal-light',
+                'hover:text-teal',
+                'text-xs'
+              )}
+            >
+              <a
+                className={classnames('px-4', 'py-2', 'flex', 'justify-between', 'items-center')}
+                href={child.url}
+                target="_blank"
+              >
+                {child.title}
+                <svg
+                  width="16"
+                  height="16"
+                  xmlns="http://www.w3.org/2000/svg"
+                  className={classnames('transform-gpu', 'scale-625')}
+                >
```

Comment:
We can avoid having to add another sizing variable for `scale` by doing something like this instead:

```
<svg
  xmlns="http://www.w3.org/2000/svg"
  viewBox="0 0 16 16"
  className={classnames('w-em', 'h-em', 'scale-75', 'transform-gpu')}
```

By specifying `viewBox` we actually scale the icon based on `em` sizing, then pop the `scale-75` default TW class on there and boom, good to go.